docs/root/start/building.rst

```diff
@@ -6,15 +6,15 @@ Building
 
 The Envoy build system uses `Bazel <https://bazel.build/>`_.
 
-In order to ease initial building and for a quick start, we provide an Ubuntu 16 based docker container that
-has everything needed inside of it to build and *statically link* Envoy, see :repo:`ci/README.md`.
+In order to ease initial building and for a quick start, we provide an Ubuntu 16 and a Windows based docker container
+that has everything needed inside of it to build and *statically link* Envoy, see :repo:`ci/README.md`.
 
 In order to build without using the Docker container, follow the instructions at :repo:`bazel/README.md`.
 
 .. _install_requirements:
 
-Requirements
-------------
+Posix Requirements
```

Comment:
perhaps Mac/Linux requirements - i think that would be more user-friendly, even if this is ~correct to spec

Reply:
hmm - just thinking there is some confusion between host and containers here

ie this is actually "linux" container in mac/linux (and kinda windows) hosts

Reply:
> just thinking there is some confusion between host and containers here

What do you mean? I think these are requirements for the host to build the envoy-static

Reply:
> perhaps Mac/Linux requirements 
+ 1

Reply:
fixed in 96eea30a8107a9eb6f17af6914a4e245fcd0fc3d